hnsendpoint.go

```diff
@@ -192,18 +192,22 @@ func (endpoint *HNSEndpoint) ContainerHotDetach(containerID string) error {
 	return modifyNetworkEndpoint(containerID, endpoint.Id, Remove)
 }
 
-// ApplyACLPolicy applies Acl Policy on the Endpoint
-func (endpoint *HNSEndpoint) ApplyACLPolicy(policy *ACLPolicy) error {
+// ApplyPolicy applies a set of ACL Policies on the Endpoint. This will
+// overwrite any existing policies which were previously applied
+func (endpoint *HNSEndpoint) ApplyACLPolicy(policies []*ACLPolicy) error {

```

Comment:
In order to maintain back-compat (just in case) and add the ability to use multiple policies, I think this could be:
```
func (endpoint *HNSEndpoint) ApplyACLPolicy(policies ...*ACLPolicy) error {
```

Reply:
Sure, sounds good. I can guarantee that no-one is calling this func today ;) but I like the flexibility that your suggested syntax provides for callers. Thanks.